unicon_backend/evaluator/tasks/programming/steps.py

```diff
@@ -215,7 +216,7 @@ def run(
             subgraph_node_ids | node_ids_to_exclude
         )
 
-        program: Program = []
+        program: list[Program | str] = []
```

Comment:
does `program: Program` mess up type checking? `Program` is already a recursive type that is equivalent to `list[Program | str]`

Reply:
it does, after changing Program to use Sequence i would get errors related to program has no methods like append/extend